app/resources/1.1/basic/shacl/basic.shacl.ttl

```diff
@@ -11,94 +11,197 @@
+      [ sh:class haObjId:Acquisition_number ]
+      [ sh:class haObjId:Alternative_number ]
+      [ sh:class haObjId:Analoge_drager ]
+      [ sh:class haObjId:Ardome ]
+      [ sh:class haObjId:Basis ]
+      [ sh:class haObjId:Historical_carrier ]
+      [ sh:class haObjId:Historical_record_number ]
+      [ sh:class haObjId:Inventarisnummer ]
+      [ sh:class haObjId:MEDIA_ID ]
+      [ sh:class haObjId:Object_number ]
+      [ sh:class haObjId:Priref ]
+      [ sh:class haObjId:Word_ID ]
+      [ sh:class haObjId:Topstuk_ID ]
+      [ sh:class haObjId:WorkPID ]
+      [ sh:class haObjId:DataPID ]
+      [ sh:class haObjId:Bestandsnaam ]
+      [ sh:class haObjId:Api ]
+      [ sh:class haObjId:Pdf ]
+      [ sh:class haObjId:PersistenteURI_Record ]
+      [ sh:class haObjId:PersistenteURI_Werk ]
+      [ sh:class haObjId:PersistenteURI_VKC_Record ]
+      [ sh:class haObjId:PersistenteURI_VKC_Werk ]
+      [ sh:class haObjId:MEEMOO-PID ]
+    );
+  ],
   [
     a sh:PropertyShape ;
     sh:path relSubType:isr ;
     sh:nodeKind sh:IRI ;
     sh:class premis:Representation ;
+    # sh:node <#RepresentationShape> ;
     sh:name "is represented by"@en ;
     sh:description "The representation of the intellectual entity."@en ;
     sh:minCount 1 ;
-    sh:maxCount 1 ;
+    sh:severity sh:Violation ;
+    sh:message "No representation or mismatch between IE's `objectIdentifier` and Representation's `objectIdentifier` and their relationships. Please check the "@en ;
   ],
   [
     a sh:PropertyShape ;
-    sh:path dct:title ;
+    sh:path dcterms:title ;
     sh:nodeKind sh:Literal ;
     sh:name "has title"@en ;
     sh:description "The title of the intellectual entity."@en ;
     sh:minCount 1 ;
+    sh:severity sh:Violation ;
+    sh:datatype rdf:langString;
+    sh:message "Intellectual entity has no `dcterms:title`."@en ;
+  ],
+  [
+    a sh:PropertyShape ;
+    sh:path dcterms:title ;
+    sh:nodeKind sh:Literal ;
+    sh:name "has title"@en ;
+    sh:description "The title of the intellectual entity."@en ;
+    sh:severity sh:Violation ;
     sh:uniqueLang true ;
     sh:datatype rdf:langString;
+    sh:message "Multiple values for `dcterms:title` with the same `@xml:lang` attribute."@en ;
   ],
+  #? alternative mag volgens SIP enkel meerdere keren gebruikt worden met verschillende Lang tag, klopt dit?
   [
     a sh:PropertyShape ;
-    sh:path dct:alternative ;
+    sh:path dcterms:alternative ;
     sh:nodeKind sh:Literal ;
     sh:datatype rdf:langString;
     sh:name "has alternative title"@en ;
     sh:description "An alternative title of the intellectual entity."@en ;
   ],
   [
     a sh:PropertyShape ;
-    sh:path dct:extent ;
+    sh:path dcterms:extent ;
+    sh:nodeKind sh:Literal ;
+    sh:name "has extent"@en ;
+    sh:description "The extent of the intellectual entity."@en ;
+    sh:maxCount 1 ;
+    sh:severity sh:Violation ;
+    sh:message "More than one value for `dcterms:extent`."@en ;
```

Comment:
Extent is volgens de SIP spec niet verplicht, maar mag wel niet meer dan 1 waarde hebben. Vandaar ook violation hier.